lms/templates/navigation.html

```diff
@@ -124,15 +124,21 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
         % endif
       </%block>
       % if not settings.FEATURES['DISABLE_LOGIN_BUTTON']:
+          % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
```

Comment:
@mattdrayer Is it fair to punt these changes over to another story / PR? I think all of our feature flag checks are done as is currently. I'm happy to draft that story if that works for you. 

Additionally, are you the second reviewer on this or were you just passing through providing useful feedback? I wanted to check in since this PR needs a second reviewer. cc'ed @pbaruah. 

@dsego was originally going to be a reviewer but as he contributed to the PR it might make sense for someone else to do a review of that work.